src/SFA.DAS.Commitments.Application/Commands/DeleteCommitment/DeleteCommitmentCommandHandler.cs

```diff
@@ -150,5 +151,13 @@ private async Task PublishV2Events(Commitment commitment)
                 await _v2EventsPublisher.PublishApprenticeshipDeleted(commitment, apprenticeship);
             }
         }
+
+        private async Task PublishMessageIfChangeOfPartyCohortRejected(Commitment commitment, CallerType caller)
+        {
+            if (caller == CallerType.Provider && commitment.ChangeOfPartyRequestId.HasValue)
```

Comment:
If it is CoE request & provider deletes the cohort, is it still going to send an email?